R/utils.R

```diff
@@ -0,0 +1,45 @@
+#' Check Recode List
+#'
+#' @param recodeList A recode list.
+#' @noRd
+#'
+#' @return NULL
+#'
+#' @examples
+#' # tbd
+checkRecode <- function(dat) {
```

Comment:
`dat` should probably be `recodeList`, to also match the argument name in the documentation? Conceptually, I prefer the argument `recodeList` as it more accurately describes what object type is given to the function.

Also: Should `checkRecodeList()` have an additional argument named, e.g. called `argumentName`, that is specified when other functions call `checkRecodeListe()` for different argument names than `recodeList()`?

Reply:
Yes makes sense, added.

Reply:
I think this was misleading wording chosing by me. What I meant was an additional argument that specifies what the `recodeList` is called in all error messages.

For instance, if `updateRecodeDB()` calls `checkRecodeList()` twice, the user does not know which `recodeList` the error message will refer to. I think there are two approaches for this issue:
- via an additional argument as suggested above (e.g., called `argumentName` such as here https://github.com/beckerbenj/eatGADS/blob/master/R/check_functions.R) 
- via automatically retrieving the argument name, as packages such as checkmate do (we will discuss this in our next meeting anyway)

Reply:
removed columnNames argument and used `checkmate` for the error messages where sensible (it really is quite elegant, so definetly worth implementing in `eatTools`).